providers/zdf/zdf.go

```diff
@@ -0,0 +1,25 @@
+package zdf
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("zdf", &Provider{})
+}
+
+// Provider merely implements the Provider interface.
+type Provider struct{}
+
+// BuildURI generates a search URL for ZDF.
+func (p *Provider) BuildURI(q string) string {
+	return fmt.Sprintf("https://www.zdf.de/suche?q=%s&synth=true&sender=Gesamtes+Angebot", url.QueryEscape(q))
+}
+
+// Tags returns the tags relevant to this provider.
+func (p *Provider) Tags() []string {
+	return []string{"television"}
```

Comment:
`tv` here too.

Reply:
Also since these are german tv, we generally will make sure to only tag them for german! So we should update this to return no tag unless the user is german, we can just check the region.

Reply:
For instance we did this for conditional chinese

```
// Tags returns the tags relevant to this provider.
func (p *Provider) Tags() []string {
	switch providers.Language() {
	case "zh":
		return []string{"forums"}
	default:
		return []string{}
	}
}// Tags returns the tags relevant to this provider.
func (p *Provider) Tags() []string {
	switch providers.Language() {
	case "zh":
		return []string{"forums"}
	default:
		return []string{}
	}
}
```